Firestore/core/test/firebase/firestore/immutable/array_sorted_map_test.cc

```diff
@@ -158,26 +138,6 @@ TEST(ArraySortedMap, BalanceProblem) {
   ASSERT_SEQ_EQ(Pairs(Sorted(to_insert)), map);
 }
 
-// TODO(wilhuff): Iterators
-
-// TODO(wilhuff): IndexOf
-
-TEST(ArraySortedMap, AvoidsCopying) {
```

Comment:
What's the reason to delete this test?

Reply:
This test was written back when `ArraySortedMap::insert` returned `std::shared_ptr`. Back then it was possible to detect that the identity of the map did not change after insertion.

Now that `ArraySortedMap` and the others have changed to returning a value-typed wrapper that internalizes the cheap copies it's no longer possible to distinguish that the identity hasn't changed this way.

This test survived that transition because all I really did was change the typedef for `IntMap` and the test happened to continue working even though it was now testing equality of the map rather than the identity.